core/src/main/resources/defaults/blueprints/7.2.10/cdp-data-engineering-ha.bp

```diff
@@ -496,7 +496,7 @@
         ]
       },
       {
-        "refName": "manager",
+        "refName": "main",
```

Comment:
Can we use a name like 'mangerGroup02'? Will make it a little more consistent if we add additional manager service hostGroups in the future (e.g. HS2/HMS). 'main' doesn't convey a lot.

Reply:
manager name was used because CM was being installed on this as per Ganesh. But if we see the services in this hostGroup there isn't anything to link it to manager.

Reply:
Sorry. I meant masterGroup02. Yes, this is not a 'manager' (CM), and 'main' is not very helpful.

Reply:
Since the hostgroup name is also part of endpoint, the hostgroup name cannot be very big. Some services like hue require max hostgroup name of 7 characters. Also endpoint generated based on masterGroup02 would be something like https://<cluster>mastergroup020, https://<cluster>mastergroup021..... since number index is used to refer node. So dont think hostGroup name should contain numbers to avoid confusion between hostgroupname and nodeIndex.  I tried various other 6-7 letter words but main seemed better than most other options.  If you any other word in mind without numerals and length of upto 6 to 7 characters  I can change to that.

Reply:
I don't have a good name. 'masterx'? That's what LinkedIn ended up using. 'main' would confuse me ... Have master01, master02, and main01 vs mastere01, master02, masterx01 | The second at least says that these are the same category of nodes, running master services.

Reply:
Auxiliary is the name used by datalake HA. I am fine with masterx, and vote for it. Using the same name as datalake HA, could confuse some folks coming from a different perspective if the intentions of the name are slightly different.

Reply:
ok will change this hostgroup name to masterx.